devops/provision/Dockerfile

```diff
@@ -60,17 +60,15 @@ RUN /virtualenv/env3/bin/pip install ubelt
 RUN . /virtualenv/env3/bin/activate \
  && pip install scikit-build
 
+# Install apt dependencies for flann
+RUN apt-get install -y liblz4-dev libhdf5-serial-dev
```

Comment:
Please add this to base so that it benefits from `&& rm -rf /var/lib/apt/lists/*`

Reply:
Also, base is the stage that `install` uses as its starting layer, so it needs to be there.

Reply:
I moved it to base.

```diff
$ git diff origin/update-dockerfiles..
diff --git a/devops/base/Dockerfile b/devops/base/Dockerfile
index 2b525cf5b..877eed7ae 100644
--- a/devops/base/Dockerfile
+++ b/devops/base/Dockerfile
@@ -64,6 +64,8 @@ RUN apt-get update \
         gfortran=4:7.4.0-1ubuntu2.3 \
         tesseract-ocr=4.00~git2288-10f4998a-2 \
         tesseract-ocr-eng=4.00~git24-0e00fe6-1.2 \
+        liblz4-dev=0.0~r131-2ubuntu3 \
+        libhdf5-serial-dev=1.10.0-patch1+docs-4 \
  && rm -rf /var/lib/apt/lists/*
 
 RUN curl -fsSL https://get.docker.com -o get-docker.sh \
diff --git a/devops/provision/Dockerfile b/devops/provision/Dockerfile
index 449a7304a..d1020d987 100644
--- a/devops/provision/Dockerfile
+++ b/devops/provision/Dockerfile
@@ -60,9 +60,6 @@ RUN /virtualenv/env3/bin/pip install ubelt
 RUN . /virtualenv/env3/bin/activate \
  && pip install scikit-build
 
-# Install apt dependencies for flann
-RUN apt-get install -y liblz4-dev libhdf5-serial-dev
-
 # Build Python repositories with external codebases
 RUN . /virtualenv/env3/bin/activate \
  && cd /wbia/flann \
```